README.md

```diff
@@ -1,5 +1,32 @@
-# TODO
+# Synced with [`microbundle@v0.12.4`](https://github.com/developit/microbundle/compare/master...eik-lib:main?w=1)
 
-- document why we forked it.
-- keep in sync in the readme what version of microbundle we're in sync with (v0.12.4 at the time of the fork).
-- document how it's intended to be used with eik, in the two target scenarios: 1) bundling a podlet. 2) bundling a library meant to be published to both an eik cdn and to npm.
+[Why did we fork microbundle?](https://eik.dev/docs/mapping_bundling#why-we-forked-microbundle)
+
+This fork is currently serving two purposes:
+
+1. Support import maps using the eik rollup plugin and other functionality that's only possible by forking, as microbundle doesn't allow extending this from the outside in its public API.
+2. Provide optimal defaults for how FINN.no is using Eik.
+
+We see the error of our ways and in the spirit of making upstream contributions to microbundle as easy as possible, we'll refactor this fork to only serve purpose 1. We'll create a new package on our GHE for FINN.no that will wrap this fork and implement the features that are useful to us but perhaps not to non-FINN use cases.
+The long term plan is that we want to delete this fork, and make our internal microbundle wrapper simply wrap microbundle directly instead of our fork.
+
```

Comment:
```suggestion
We see the error of our ways and in the spirit of making upstream contributions to microbundle as easy as possible, we'll refactor this fork to only serve purpose 1. We'll create a new package on our GHE for FINN.no that will wrap this fork and implement the features that are useful to us but perhaps not to non-FINN use cases.
The long term plan is that we want to delete this fork and make our internal microbundle wrapper simply wrap microbundle directly instead of our fork.

```